### DISCUSSION TITLE
Are you comimg to RSS 2023??

### DISCUSSION THREAD

**Discussion by psh9002** [Toni-SM/skrl#86] [General]
Hi, @Toni-SM 

Excuse me but, are you comimg to [RSS 2023](https://roboticsconference.org/)??  It will be held in Daegu, Korea from July 10 to July 14. I think you've helped me a lot, so I just want to say hello if you will come.

(If you think this is too trivial, feel free to delete it.)

**Comment by Toni-SM** [comment]
Hi @psh9002 

Thank you for your kind words and the information.
I will not be in the RSS 2023. Although it would be nice to be able to attend online to learn about the advances in robotics. 
Do you know of a way to attend the event that doesn't require registration?

**Reply by psh9002** [reply]
Hi, @Toni-SM 

I searched for workshops and several sections but, there is only $50 virtual registration.

**Reply by psh9002** [reply]
You can watch the presentations on YouTube. (https://roboticsconference.org/)
